cmake/ConfigFileSetting.cmake

```diff
@@ -263,6 +264,17 @@ nrn_configure_file(nmodlconf.h .)
 nrn_configure_file(nrnunits.lib share/lib OUTPUT share/nrn/lib)
 nrn_configure_file(nrn.defaults share/lib OUTPUT share/nrn/lib)
 nrn_configure_file(constants.py share/lib/python/neuron/rxd)
+if (MINGW)
+  set(dos_marshal_dir "c:\\marshalnrn")
```

Comment:
nitpick : if there are any variables, I typically use upper case. (Is it `dos`? or `win` better?)

Reply:
Would you change this given that it would also have to be changed in two other files as well--- configure.ac and nrnset6upmingw.nsi.in ? Nevertheless, I have changed to NRN_MARSHAL_DIR